packages/make-styles-webpack-loader/src/webpackLoader.ts

```diff
@@ -28,6 +33,13 @@ export function webpackLoader(
   sourceCode: WebpackLoaderParams[0],
   inputSourceMap: WebpackLoaderParams[1],
 ) {
+  const options = getOptions(this) as WebpackLoaderOptions;
+
+  validate(configSchema, options, {
+    name: '@fluentui/make-styles-webpack-loader',
+    baseDataPath: 'options',
+  });
```

Comment:
Usage of `loader-utils` & `schema-utils` is suggested in Webpack docs, https://webpack.js.org/contribute/writing-a-loader/#loader-utilities.